### ISSUE TITLE
Just some positive feedback to the team

### ISSUE THREAD

**Issue by amityweb** [freescout-helpdesk/freescout#233]
I just want to say we have been using Freescout for a few weeks now and so far its brilliant. Its been working well. So just want to say thank to the team for this system, its very good. 

I can see a modules section, I assume at some point some paid modules will come along, so happy to contribute buy buying some modules if I need them at some point.

One idea for a module would be Asana integration. So any new ticket cerated is added into an Asana project we choose, and it includes the body of message and link back to the freescout conversation. We used to do this in Helpscout using Zapier, but since they introduced pricing plans, the API was not even in their lowest cost plan, we would have had to pay even more per user (?) per month to get Zapier working (which also costs). So our entire cost for 3 systems was very high. 

Otherwise is Asana is too specific, then maybe Zapier or IFTTT so we can submit them into various systems that IFTTT and Zapier support (rather than specific modules for each 3rd party system). 

So I look forward to more functionality added and more modules we can use. 

Thanks, great product.

**Comment by tareko** [comment]
+1 to the positive feedback. our team has been using FS for the past month or so and it has been great! We came from no helpdesk solution since we have a Free/Open Source requirement for our project stack.

tarek : )

**Comment by franckadil** [comment]
@freescout-helpdesk Do you plan to add webhooks/zapier support for easy integrations in users workflow ?

**Comment by freescout-helpdesk** [comment]
> @freescout-helpdesk Do you plan to add webhooks/zapier support for easy integration in users workflow ?

Yes, webhooks and zepier integration will be implemented as modules.

**Comment by franckadil** [comment]
@freescout-helpdesk looks promising 🥇  We will be happy to contribute!

**Comment by sam-suresh** [comment]
Thank you for this Awesome work! This is really a great app.
Looking forward for paid modules to fund the project and keep it going. 

Please add 2FA when possible.

**Comment by johnlund** [comment]
We just installed FreeScout and it is a life-saver. Thank you so much! We'll be purchasing some modules and looking forward to further integrations. :)

**Comment by Zatalyz** [comment]
Thank you for developing this application on AGPL. It is much easier to use than other open source solutions. It is really wonderful. It will help my association. I am working to improve French translation, I hope I push it soon !

**Comment by benninkcorien** [comment]
Same. Painless install. Will explore some modules tomorrow, but really happy so far!

**Comment by amirkhan7** [comment]
same as above, loving freescout so far. we moved over from redmine (which we were using as a helpdesk) and our users love the interface and simplicity!

**Comment by diguinho2018** [comment]
Yes, congratulations guys on this great app.
After a long time searching and testing a lot of app/plugins to give my users a great Customer Support, I'm glad I found FreeScout.
I'm going to buy almost all modules to support the team and to increase the functionality of the FreeScout.

Well done guys, looking forward to more awesome features and modules.

**Comment by sylvia83** [comment]
You guys do a great job and are very responsive. We buy modules just to support and encourage others to do the same!

**Comment by endoflevelboss** [comment]
+1

Great tool, great code, great style. We were looking for a free and self-hosted email management tool for a long time and found freescout only now. It is a pleasure to work with the unobtrusive yet powerful user interface. This seems to be unique in a market of bloated, old-fashioned or overdone, social media focussed and overpriced services, hosted on someone else's systems. Prices for the modules are ridiculous compared to the great added value. Could easily be a multiple while still being worth the money.

**Comment by jensolsson** [comment]
Also love it so far! The GUI has a lovely simplistic design and feel very modern and robust. Great job!

**Comment by ecomturbo** [comment]
Really solid must have App. Congrats @freescout-helpdesk

**Comment by jesperordrup** [comment]
Freescout is really a great solution. Clean UI. Keep it up. Bought most modules already. And it's really working out well. Im sorry we cant contribute to codebase since we have zero php, laravel experience but I'll make sure to support it in another way.

Amazing work @freescout-helpdesk and contributers. Thanks

**Comment by pentanet** [comment]
Hey Freedesk dev/s. Looking at the positive sentiment I  too would love to support this project, maybe even have our team contribute to it (we are a Laravel shop), but the total price for the modules that interest us is around US$250, and that is too much to pay to try the software out. If the software suits us, then of course supporting the project is a non-brainer.

Please consider adding a time-based trial license that unlocks all modules for a week or something to allow users to evaluate it. I note that you previously did but removed it due to abuse.

**Comment by woolmilkpigs** [comment]
Good work! I love the simple UI but powerful software behind! Thank you!

**Comment by jake-the-just** [comment]
Would like to thank you. You implement feedback pretty quickly, just great. Thank you!

**Comment by arborrow** [comment]
Greetings, I am working with the team at [Jesuit Refugee Services](https://jrs.net/) to explore options for a ticketing system and they like what they see. I noticed that the project is currently on Laravel 5 and I was wondering if there was any intention or interest in getting it upgraded to Laravel 9. From a package security perspective I would think keeping things updated might be helpful. I look forward to exploring the project.

**Comment by ar-ml** [comment]
I compared about 10 different ticketing solutions, most of them cost at least 15USD per month PER AGENT!!!
**freescout ist by far the best solution if found!** 
Maybe Installing it could be easier (after wasting one day in the SSH terminal I had to had to hire a linux admin who did it  within one hour).
But after that I was very happy with the setup process in freescout itself, and also with the UI, the great functionality, the modules and the fair business model / moderate onetime fees for the modules - even string translation is possible from within the program!

@freescout-helpdesk Thanks for your great work!

**Comment by onderzoekspraktijk** [comment]
I think your [1.8.51] release wonderfull!

This release FIXES TWO MAJOR ISSUES which existed since the beginning of FreeScout (see "Fixed" section below).
- an issue with the background job (queue:work) not being executed automatically and executing only after clearing the cache.
-  Make sure fetching is executed even if the previous fetch command was killed or threw an error and did not remove the mutex.

For years now my workflow was to check and double check the queue was again not working, to flush cache, trigger the special link to get it going again, etc.
Like an old steam engine where you have to always kick it to start it. I only now accept it really is not necessary to do this routine.

That is over now! Thank you.

Cheers,
Paul

**Comment by AlecKinnear** [comment]
Nice to see the positive reactions here to FreeScout. Since we discovered FreeScout three years ago, we've contributed $50/month every month via Patreon. That's the minimum we'd pay for a commercial solution, which would come with all kinds of limitations and which would be far more difficult to customise. We're a small company so this is a significant contribution for us.

I invite other commercial users of FreeScout to join us in supporting FreeScout on a monthly basis or even with a single large donation. I suggest the monthly option as long as you/we are using FreeScout as monthly keeps a developer's spirits up more over time and provides long-term motivation to keep going.

**Comment by saiftheboss7** [comment]
We love FreeScout. Love for the team. 

One suggestion would be upgrading to Laravel 10 at some point in the future!

**Comment by cashycash** [comment]
♥️ to the team

**Comment by steveb85** [comment]
+1 all around nice work

**Comment by marianstancu** [comment]
Great, GREAT job, guys, I'm impressed!

**Comment by LK-ML** [comment]
Thank you for Freescout 🙏💝

**Comment by Mario-Greiner** [comment]
I've been using freescout for a few weeks now and I have to say that the team has done a great job.
It's very easy to set up and use.
I also like the model with the so-called “modules”, which basically means that it's free to use and if you want some additional features, you can support the work and add functionality to the system.
The only thing I am missing is that you cannot customize the URLs in the Knowledgebase and End-UserPortal. Should this be implemented, I would describe the Freescout system as PERFECT.

Love and respect for the amazing job 🫶

**Comment by JamesMoo-it** [comment]
@amityweb Are you guys still using it?

**Comment by fisthandle** [comment]
I really like freescout. I've bought 6 modules so far (two more today to support you) and have been developing my own module for my needs. App is great, customizable, easy to use and more powerful than i thought.  I use it every day, almost 4k tickets in the last month.

It works well and made supporting our customers a better, faster, cleaner experience. Respect!

**Comment by joris85** [comment]
I am using Freescout for many years for multiple organizations where I am part of. 
I have bought most of the modules and it I am getting more and more enthousiastic about it :-)

**Comment by czottmann** [comment]
Thanks for the project, folks! Good stuff. I've spent some money on about a dozen modules, and thanks to API and FS's general flexibility, I was able to migrate successfully from Help Scout to my semi-self-hosted FS instance. Singing your praises already.

And I like that the modules system makes it quite easy to create [my own modules](https://github.com/czottmann/freescout-extrajs). 👨🏼‍🍳👌🏼

Keep up the good work!

**Comment by TurboJens** [comment]
This is great! I was lucky enough to discover Freescout a few months ago and after testing it on my own mailbox I was able to convince my collegues that we had to use this instead of email.

I can hardly find any negetive about Freescout and the the improvement in workflow compared to a mix of email and phone calls is amazing - and that's even before we really started since it takes time to educate customers to use this new feauture (support email). I have a timeline where I want to include more mailboxes for teams working with sales too. 

So keep ut på the good work! I have loaded a huge amount of modules, and I really like the pricing for this, it's not expensive and it's not a recurring cost so it's very easy to try and like! :)

And it's lightning fast and easy to use too 🥇

**Comment by bmgtr** [comment]
+1 Awesome software. The idea with individually paid features is just awesome!

Guys, show your love in _public_, eg. at [AlternativeTo.net](https://alternativeto.net/software/freescout/about/)

**Comment by AdamCoffeeOverflow** [comment]
Jumping into this for 2026, agreed very underrated! love everything about it so far

**Comment by teamrehab** [comment]
A heartfelt thank you for all the bugs/features you've triaged for me!

**Comment by schenardi** [comment]
Freescout is a great tool - we're a non commercial administration for associations and use it as central place to manage all mailboxes and got many positive feedbacks.
